galeracluster/source/documentation/galera-status-variables.rst

```diff
@@ -787,7 +828,7 @@ To retrieve the value of this status variable, execute the ``SHOW STATUS`` state
 
 .. code-block:: mysql
 
-   SHOW STATUS LIKE 'wsrep_flow_control_paused';
+   SHOW STATUS LIKE 'wsrep_flow_control_active';

```

Comment:
but wsrep_flow_control_paused is valid no? why did this change?

Reply:
I changed it because it was mentioned incorrectly in the context:

![image](https://github.com/user-attachments/assets/6d8c5e05-01a9-42e7-91e2-9e572d7855a9)

Reply:
good catch then